Sources_v3/StreamChatUI/Common Views/ChatSquareButton.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class ChatSquareButton: Button {
+    open var defaultSize: CGFloat?
+    
+    public init(image: UIImage? = nil, tintColor: UIColor? = nil) {
```

Comment:
Why do you pass in the tint color in the initializer when the button already has a tint color? Tint color can change and depends on the superview's tint color.